README.md

```diff
@@ -1 +1,2 @@
-# cloudformation-templates
\ No newline at end of file
+# cloudformation-templates
+This repository contains public cloudformation templates for logzio customers to automate aws integrations deployments
```

Comment:
Let's add here:
1. List of templates in this folder.
2. Tagging instructions for releases
3. Instructions to deploy the stack (also needs to be in logzio's docs)